composer.json

```diff
@@ -47,13 +47,13 @@
         }
     },
     "autoload": {
-        "psr-0": {
-            "Knp\\Component": "src"
+        "psr-4": {
+            "Knp\\Component\\": "src/Knp/Component"
         }
     },
     "autoload-dev": {
-        "psr-0": {
-            "Test": "tests"
+        "psr-4": {
+            "Test\\": "tests/Test"
```

Comment:
Maybe `Knp\Component\Tests`?

Reply:
It's how it's done in Symfony components for example https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/EventDispatcher/Tests/EventDispatcherTest.php#L12

Reply:
That is the old namespace and I kept it just to minimize changes. If you think it's important, I can change it.

Reply:
oh okay, not important :+1: